src/model/statsd.coffee

```diff
@@ -0,0 +1,173 @@
+http = require 'http'
```

Comment:
I don't think this file should be stored in src/model/ it doesn't really have anything to do with the model. It's more of a set of functions that are called for API. Perhaps, this should rather be moved to src/api/.